all.json

```diff
@@ -28,6 +28,10 @@
 	],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"fastcoiner.com",
+		"fixedfloat.to",
```

Comment:
Not convinced on this one.

Reply:
which one? Fastcoiner is a generic seedstealer and fixedfloat.to is the fake version of fixedfloat.com

Reply:
The issue with the fixedfloat item is that from the screenshots and the urlscan there is no definitive proof that it is less than honest. So any additional info here would be great.

Reply:
There are over 10 submits like this with the exact same screenshot

Reply:
This is a very well known scam. The real one is fixedfloat.com